build/config/config1.json

```diff
@@ -22,11 +22,11 @@
     "block_store_path": "/var/blockstore",
     "tx_indexer_path": "",
     "trees_store_dir": "/var/trees",
-    "max_conns_count": 8,
-    "min_conns_count": 0,
-    "max_conn_lifetime": "1h",
-    "max_conn_idle_time": "30m",
-    "health_check_period": "5m"
+    "max_conns_count": 50,
+    "min_conns_count": 1,
+    "max_conn_lifetime": "5m",
+    "max_conn_idle_time": "1m",
+    "health_check_period": "30s"
```

Comment:
There's inconsistency between these values and `build/localnet/manifests/configs.yaml` is that on purpose?